examples/README.md

```diff
@@ -28,7 +28,12 @@ For example: `dotnet run -- --server https://my_server admin "my password"`.
 Examples
 --------
 
-### Configuration
+### Common
```

Comment:
Question about replacing "configuration" with "common": Will this confuse users about which API they need to use?

Reply:
I wasn't sure what to really put as the header. My thinking was calling it "Common" would reinforce that the configuration stuff applies across all of the C# APIs and isn't specific to any one. Whereas saying Configuration at the same level as the other APIs could imply there's a Configuration API that's separate from the others.

I guess another way would be explicit about it, like have two sections:

Common Examples
---------------

The following examples demonstrate concepts common across all SystemLink client APIs:

- Configuration ...

API Examples
------------

### Message
...

Reply:
Ah! Your explanation makes sense, and I like your idea for making it more explicit.